cloudify_marketplace/provision/upload_blueprint

```diff
@@ -1,15 +1,24 @@
 #! /usr/bin/env bash
-CFY_BIN=~/cfy/bin/cfy
+CFY_BIN=/opt/cfy_cli/bin/cfy
 cd ${BLUEPRINT_PATH}
 
 # Inject the correct username
 sudo sed -i "s/<<IMAGEBUILDERUSER>>/$(whoami)/" scripts/configure.py
 
+# Use the correct credentials
+export CLOUDIFY_USERNAME=${CLOUDIFY_MANAGER_USERNAME}
```

Comment:
Should the credentials bit be under the CLOUDIFY_MANAGER_SECURITY_ENABLED gate? It won't actually break anything at the moment (since it just sets an auth header that is ignored), but it would be cleaner and avoid unexpected breakage in the future if the auth method changes.